package.json

```diff
@@ -171,5 +180,8 @@
     "webpack": "^3.8.1",
     "webpack-dev-middleware": "^3.1.3",
     "webpack-hot-middleware": "^2.22.1"
-  }
+  },
+  "pre-commit": [
+    "test"
+  ]
```

Comment:
Not sure about running tests before every commit... I can see that getting in the way while developing. It should be enough if it fails the build on CI.

Reply:
I agree... I've commented here https://github.com/elifesciences/elife-xpub/issues/1462#issuecomment-463545996

Reply:
we can do it in pre-push, but I think is necessary before to upload new code, to run the tests and be sure there are no tests broken

Reply:
removed pre-commit, with this config, now the test will be executed on the CI, and will fail if the coverage decrease, from the one defined in the threshold, which matches with our current coverage.
the idea behind this is to update the threshold config periodically, so will ensure the coverage increases